kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/execution/ArgumentTransformer.kt

```diff
@@ -26,7 +26,8 @@ open class ArgumentTransformer(val schema : DefaultSchema) {
                 }
             }
             value is ValueNode.ObjectValueNode -> {
-                val params = type.unwrapped().kClass!!.primaryConstructor!!.parameters.associateBy { it.name }
+                val constructor = type.unwrapped().kClass!!.primaryConstructor ?: throw GraphQLError("Java class as inputType are not supported")
```

Comment:
As we do have access to the specific `ValueNode` here, you should provide that to the `GraphQLError` exception also: `throw GraphQLError("...", value)`

It would also be great if you could provide the java class name in the exception message 👍 
As we do have access to the class, it would be great to include the name of the Java class in this error.

Reply:
Done 👍